torch/_subclasses/functional_tensor.py

```diff
@@ -346,6 +357,17 @@ def __init__(self, pre_dispatch=False, export=False, _allow_token_discovery=Fals
             torch.storage.UntypedStorage, Optional[FunctionalTensor]
         ] = weakref.WeakKeyDictionary()
 
+        # Used for export only.
+        # Map from FunctionalTensor's storage to a group id.
```

Comment:
Just a nit, but this description doesn't tell me much (what is a group id?).

What do you think of something like this?
```
During export, we want to treat all potentially-aliased tensors that come from `aten.to` as part of a single "aten.to group",
so that we can properly raise an error when any members of the group are mutated.
Whenever a tensor (or its alias) participates as an input/output to `aten.to`, we assign its "aten.to group" a unique id.
```